backend/capellacollab/alembic/versions/cf93aadf77d6_add_t4c_instances.py

```diff
@@ -37,7 +37,6 @@ def upgrade():
             ["versions.id"],
         ),
         sa.PrimaryKeyConstraint("id"),
-        sa.UniqueConstraint("name"),
```

Comment:
I'd say it makes sense, right? When linking the a t4c model, the instance name is the unique identifier, right? Otherwise the user can not see the difference.

Reply:
Yes you’re right, I had this reflexion when thinking about the fact that an instance could be duplicated to support several protocols, but actually we should still be able to differenciate them by the name (even if it means adding a precision).

Reply:
Can you add it back then?

Reply:
Of course, I was working on another PR.